train.py

```diff
@@ -708,6 +716,8 @@ def main():
             objective,
             n_trials=NUMBER_OF_STUDY_TRIALS,
             callbacks=[TerminatorCallback(terminator)],
+            n_jobs=4,
```

Comment:
We don't use that as it uses threads and that runs into the GIL. Better to use multiple processes